packages/engine/Source/Scene/ITwinData.js

```diff
@@ -0,0 +1,74 @@
+import Cesium3DTileset from "./Cesium3DTileset.js";
+import defined from "../Core/defined.js";
+import Resource from "../Core/Resource.js";
+import ITwinPlatform from "../Core/ITwinPlatform.js";
+import RuntimeError from "../Core/RuntimeError.js";
+
+/**
+ * Methods for loading iTwin platform data into CesiumJS
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ *
+ * @see ITwinPlatform to set the API token and base API URL
+ * @namespace ITwinData
+ */
+const ITwinData = {};
+
+/**
+ * Create a {@link Cesium3DTileset} for the given iModel id using the mesh export service.
```

Comment:
```suggestion
 * Create a {@link Cesium3DTileset} for the given iModel id using iTwin's Mesh Export API.
```

Pulling names and capitalization from https://developer.bentley.com/apis/mesh-export/.